frontend/encore/vuejs.rst

```diff
@@ -34,6 +34,80 @@ also configure the `vue-loader options`_ via a callback:
         };
     });
 
+Building a simple component
+---------------------------
+
+Once Vue is installed and ``webpack.config.js`` has been updated,
+let's build our first component and display a simple ``Hello World from Vue !``,
+in order to ease the process, the usage of ``.vue`` files is recommended,
+let's start by adding a ``main.js`` file inside the assets folder:
```

Comment:
This is one big long sentence. Let's split into a few. And try to avoid let's ;)